richtext/richtext.lua

```diff
@@ -634,5 +634,15 @@ function M.characters(word)
 	return chars
 end
 
+---Removes the gui nodes created by rich text
+function M.remove(words)
+	assert(words)
+
+	local num = table.getn(words)
```

Comment:
table.getn() was deprecated in 5.1. The length operator (#) is the recommended way to get the length of a table:

https://www.lua.org/manual/5.1/manual.html#7.2

Reply:
Updated to replace table.getn() with #.
Added documentation.